spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -91,6 +91,12 @@
 from spyder.utils.qstringhelpers import qstring_length
 from spyder.widgets.helperwidgets import MessageCheckBox
 
+if not PY2:
+    from IPython.core.inputtransformer2 import TransformerManager
+else:
+    from IPython.core.inputsplitter import IPythonInputSplitter as \
+        TransformerManager

```

Comment:
Please change this according to what I did in PR https://github.com/spyder-ide/spyder/pull/13721